apps/syntrest/src/main/java/no/nav/registre/syntrest/consumer/SyntGetConsumer.java

```diff
@@ -0,0 +1,51 @@
+package no.nav.registre.syntrest.consumer;
+
+import io.kubernetes.client.ApiException;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.registre.syntrest.kubernetes.ApplicationManager;
+import org.springframework.web.util.UriBuilderFactory;
+
+import java.net.URI;
+
+import static org.apache.commons.lang.StringUtils.ordinalIndexOf;
+
+@Slf4j
+public class SyntGetConsumer<T> extends SyntConsumer {
+
+    private final Class<T> clazz;
```

Comment:
Kunne for eks. kalt det `responseClass` i stedet for `clazz`. Syns det ble litt rart med `zz`.